index.js

```diff
@@ -0,0 +1,179 @@
+// import { List } from "./modules/booklist";
+
+// const list = new List();
+// list.load();
+
```

Comment:
- [ ] Please try to avoid having unnecessary lines of code, they may be very confusing when you get back to this project in a while 🤔

Reply:
Thank you